apps/site/pages/en/learn/command-line/output-to-the-command-line-using-nodejs.md

```diff
@@ -168,44 +169,31 @@ It will not appear in the console, but it will appear in the error log.
 
 ### Color the output
 
-You can color the output of your text in the console by using [escape sequences](https://gist.github.com/iamnewton/8754917). An escape sequence is a set of characters that identifies a color.
-
-Example:
-
-```js
-console.log('\x1b[33m%s\x1b[0m', 'hi!');
-```
-
-You can try that in the Node.js REPL, and it will print `hi!` in yellow.
+> **NOTE**
+> This part of the resource is designed with version 22.11 which notes `styleText` as ‘Active development’.
```

Comment:
Chalk is extremely popular and styleText is still in development. I don't have a strong opinion - but I do feel this change might be premature

Reply:
My idea behind this was to just talk about node and not the rest because the purpose of the learn section is to talk about node and not external packages.

Reply:
I think the phrasing is odd, shouldn't mention 22.11 specifically, Learn content is always designed with LTS in mind, and it is available in LTS atm.

Reply:
There is a PR to mark `styleText` as stable, so it's fine.